### DISCUSSION TITLE
How can I drop just one tcp message

### DISCUSSION THREAD

**Discussion by yeongbin-jo** [mitmproxy/mitmproxy#5857] [Q&A]
```python
def tcp_message(flow: TCPFlow):
    flow.kill()
```

The example above doesn't seem to work as I intended.

```python
def tcp_message(flow: TCPFlow):
    message = flow.messages[-1]
```

Is it possible in mitmproxy to drop only the latest messages?

**Comment by Prinzhorn** [comment]
> Note that TCP is stream-based and not message-based. For practical purposes the stream is chunked into messages here, but you should not rely on message boundaries.

https://docs.mitmproxy.org/stable/api/mitmproxy/tcp.html#TCPMessage

I _guess_ `message.content = b''` will have roughly the effect you're tying to achieve?
